### PR TITLE
Update #44 Review Responsibilities of the Research and Development Director

### LINKED ISSUE #44: Review Responsibilities of the Research and Development Director (5.C.7)
Check one:
- [ ] Semantic Issue: something about the meaning of the text is different
- [ ] Non-semantic Issue: Spelling, grammar, or formatting changes.

Summary of issue:
Talk with current and former Research and Development Directors about the accuracy of and any suggested changes to the constitutional responsibilities of the role.

### PR DESCRIPTION
**Pull request by Sam-Cordry**
Check one:
- [x] Semantic Change: something about the meaning of the text is different
- [ ] Non-semantic Change: Spelling, grammar, or formatting changes.

Summary of change(s):
Resolves #44 
Namely, this removes point (d) from the description.

### CODE REVIEW

File: constitution.tex [hunk 1]
```diff
@@ -416,7 +416,6 @@
 	\item To preside over Research and Development Meetings in which House members are encouraged to bring ideas for projects
 	\item To exercise general supervision over Research and Development operations
 	\item To oversee the planning, organization and construction of technical projects for the House's benefit
-	\item To collaborate with the Operational Communications Director in an attempt to fulfill the House's need for technical equipment
```
Comment (rhochgraf21) [inline constitution.tex:419]: Curious what prompted this change. If I recall correctly, a main role of RnD is to support members in reaching the technical equipment that will help them complete their project goals (eg, One Project Per Frosh program).

**Review by esoccoli — APPROVED** [review]
lgtm

File: constitution.tex (hunk 1 above)
Comment (esoccoli) [reply constitution.tex:419]: It doesnt seem necessary to specify that rnd should work with opcomm for technical equipment, just generally stating that rnd helps members with projects seems fine to me

Comment (rhochgraf21) [reply constitution.tex:419]: In-person discussion: "To strive to fulfill the House's need for technical equipment."

Comment (Sam-Cordry) [reply constitution.tex:419]: I added this point back into the section.

**Comment by rhochgraf21** [comment]
Period at the end of the list is inconsistent, do we want it to be all one way or another?

**Pull request merged**